### PR TITLE
Package Upgrade: Lodash.

### PR DESCRIPTION
**Pull request by PawBud**

### CODE REVIEW

**Comment by PawBud** [comment]
Not to be merged until I completely test out the companion app.

File: build/yarn.lock [hunk 1]
```diff
@@ -408,9 +408,9 @@ lodash.sortby@^4.7.0:
   resolved "https://registry.yarnpkg.com/lodash.sortby/-/lodash.sortby-4.7.0.tgz#edd14c824e2cc9c1e0b0a1b42bb5210516a42438"
   integrity sha1-7dFMgk4sycHgsKG0K7UhBRakJDg=
 
-lodash@^4.17.15:
-  version "4.17.15"
-  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.15.tgz#b447f6670a0455bbfeedd11392eff330ea097548"
+lodash@^4.17.19:
+  version "4.17.19"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.19.tgz#b447f6670a0455bbfeedd11392eff330ea097548"
```
Comment (PawBud) [inline build/yarn.lock:413]: This needs to be changed

**Comment by the-gem-dev** [comment]
okay

**Comment by PawBud** [comment]
It seems my yarn has issues, if anyone else can test this out and correct it ; I will be highly obliged

**Pull request closed without merging**